src/test/java/org/springframework/integration/kafka/listener/TestSingleBrokerExternal.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.listener;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import org.springframework.integration.kafka.rule.KafkaRule;
+import org.springframework.integration.kafka.rule.KafkaRunning;
+
+/**
+ * @author Marius Bogoevici
+ */
+@Ignore
```

Comment:
Looks like you haven't finished yet.
When I do this `AbstractBrokerTest#createTopic`: `sleep(partitionCount * 200);` all test pass, even those with `@Ignore`, excluding `testHighVolumeMediumConcurrency`, which hangs.
I think we can forget these `@Ignore` tests for now, but think more about that time window between partitions population in the Kafka and start of the real test - `runMessageListenerTest(...)`.
Doesn't Kafka really have something like `TestUtils.waitUntilMetadataIPropagated` for the standalone server?